src/transports/winhttp.c

```diff
@@ -788,14 +793,21 @@ static int winhttp_connect(
 	}
 
 	/*
-	 * Do a best-effort attempt to enable TLS 1.2 but allow this to
-	 * fail; if TLS 1.2 support is not available for some reason,
+	 * Do a best-effort attempt to enable TLS 1.3 and 1.2 but allow this to
+	 * fail; if TLS 1.2 or 1.3 support is not available for some reason,
 	 * ignore the failure (it will keep the default protocols).
 	 */
-	WinHttpSetOption(t->session,
+	if (WinHttpSetOption(t->session,
+	    WINHTTP_OPTION_SECURE_PROTOCOLS,
+	    &protocols,
+	    sizeof(protocols)) == FALSE) {
+	    protocols &= ~WINHTTP_FLAG_SECURE_PROTOCOL_TLS1_3;
+	    MessageBox(NULL, "djdjd", "kjfjf", 0);
```

Comment:
Looks like you've left some testing/debugging code in.

Reply:
Oops, thanks!

Reply:
I already wondered why the test timed out...

Thanks, fixed.